foundation-extensions/identity-for-edge-network/README.md

```diff
@@ -100,7 +111,7 @@ public class MobileApp extends Application {
 ```
 {% endtab %}
 
-{% tab title="iOS — Swift" %}
+{% tab title="iOS (AEP 3.x)" %}
 ### Swift
 
 ```swift
```

Comment:
Let's include the module for swift registration instructions: AEPEdgeIdentity.Identity.self